docs/src/component-statuses.js

```diff
@@ -0,0 +1,74 @@
+import componentMetadata from '@primer/component-metadata'
+import {Link} from '@primer/components'
+import StatusLabel from '@primer/gatsby-theme-doctocat/src/components/status-label'
+import Table from '@primer/gatsby-theme-doctocat/src/components/table'
+import {graphql, Link as GatsbyLink, useStaticQuery} from 'gatsby'
+import React from 'react'
+
+export function ComponentStatuses() {
+  const data = useStaticQuery(graphql`
+    query ComponentStatuses {
+      allSitePage {
+        nodes {
+          path
+          context {
+            frontmatter {
+              title
+              status
+              description
+              componentId
+            }
+          }
+        }
+      }
+    }
+  `)
+
+  const pages = data.allSitePage.nodes
+    // Only show components that have a status
+    .filter(node => node.context.frontmatter && node.context.frontmatter.status !== null)
+    // Sort alphabetically by title
+    .sort((a, b) => a.context.frontmatter.title.localeCompare(b.context.frontmatter.title))
+
+  return (
+    <div>
+      <Table>
+        <thead>
+          <tr>
+            <th align="left">Component</th>
+            <th align="left">Status</th>
+            <th align="left">Description</th>
+          </tr>
+        </thead>
+        <tbody>
+          {pages.map(page => {
+            // eslint-disable-next-line prefer-const
+            let {title, status, description, componentId} = page.context.frontmatter
+
+            const component = componentMetadata.components[componentId]
+
+            // Auto-populate title and description using component metadata
+            if (component) {
+              title ||= component.displayName
```

Comment:
How do you feel about `ActionList` and `ActionList v2` on the same list, should we title the draft differently? 

![image](https://user-images.githubusercontent.com/1863771/143870752-21fbaa11-80d5-420c-8e82-ff04581bd1db.png)

Reply:
> should we title the draft differently?

What did you have in mind?

Reply:
Drafts/ActionList v2?

Reply:
Hmm, I think we probably need to put more thought into how we communicate "drafts" in our docs. Any objection to merging this as-is and dealing with "drafts" as a follow up?

Reply:
Because of the timezone delay, I'm going to go ahead and merge this but feel free to ping me on Slack with any follow-up ideas.